### PR TITLE
Towards official PFNano production for early Run 3

### PR DESCRIPTION
**Pull request by colizz**
1. Change and simplify the default config for `PFnano_customizeData` and `PFnano_customizeMC`.

  Now the default config for including tagger inputs is like:
  - for AK4: keep DeepCSV (only jet-based vars), and DeepJet (pick only leading 3 nPF/cPF candidates + the truth variable)
  - for AK8: keep DDX inputs

2. Adopt the latest global tag according to https://twiki.cern.ch/twiki/bin/view/CMS/PdmVRun3Analysis

3. Add the sets of cfg with `_noPF`, as we expect not to include PFCands for samples other than QCD MC and BTagMu/JetMET data.

### CODE REVIEW

**Comment by colizz** [comment]
Note: for QCD MC and BTagMu/JetMET data, we can still include both AK4 and AK8 constituents for the following reason:

- The FatJetPFCands_ branches will always exist even if one specifies `AK4JetsOnly`, although their corresponding `PFCands_` features are removed. 
 - The FatJetPFCands_ branches can be problematic because the AK8 jets are not rerun, and the PFCands are not changed to the new one run obtained from running PUPPI. 
 - Including both AK4 and AK8 constituents will not increase the size much compared with AK4-only. The increase can be only 3% evaluated from a high pT QCD MC.

File: python/addBTV.py [hunk 1]
```diff
@@ -177,53 +177,53 @@ def get_DDX_vars():
...
-        DeepCSV_trackEtaRel_0 = Var("?tagInfo(\'pfDeepCSV\').taggingVariables.checkTag(\'trackEtaRel\')?tagInfo(\'pfDeepCSV\').taggingVariables.getList(\'trackEtaRel\')[0]:-999", float, doc="track pseudorapidity, relative to the jet axis", precision=10),
-        DeepCSV_trackEtaRel_1 = Var("?tagInfo(\'pfDeepCSV\').taggingVariables.checkTag(\'trackEtaRel\')?tagInfo(\'pfDeepCSV\').taggingVariables.getList(\'trackEtaRel\')[1]:-999", float, doc="track pseudorapidity, relative to the jet axis", precision=10),
-        DeepCSV_trackEtaRel_2 = Var("?tagInfo(\'pfDeepCSV\').taggingVariables.checkTag(\'trackEtaRel\')?tagInfo(\'pfDeepCSV\').taggingVariables.getList(\'trackEtaRel\')[2]:-999", float, doc="track pseudorapidity, relative to the jet axis", precision=10),
-        DeepCSV_trackEtaRel_3 = Var("?tagInfo(\'pfDeepCSV\').taggingVariables.checkTag(\'trackEtaRel\')?tagInfo(\'pfDeepCSV\').taggingVariables.getList(\'trackEtaRel\')[3]:-999", float, doc="track pseudorapidity, relative to the jet axis", precision=10),
+        # Track based (keep only jet-based features for DeepCSV from Run 3 commissioning)
```
Comment (demuller) [inline python/addBTV.py:180]: @coli is there a reason why the jet-based DeepCSV vars need to be kept? Are they needed by some SF teams?

Comment (colizz) [reply python/addBTV.py:180]: Yes, these are needed by the SF teams

**Review by demuller — APPROVED** [review]
Hi @colizz thanks a lot for your PR, it looks fine to me :)

**Pull request merged**